packages/worker/src/migrations/index.ts

```diff
@@ -0,0 +1,74 @@
+import { migrations, redis } from "@budibase/backend-core"
+import { Migration, MigrationOptions, MigrationName } from "@budibase/types"
+import env from "../environment"
+
+// migration functions
+import * as syncUserInfo from "./functions/globalInfoSyncUsers"
+
+/**
+ * Populate the migration function and additional configuration from
+ * the static migration definitions.
+ */
+export const buildMigrations = () => {
+  const definitions = migrations.DEFINITIONS
+  const workerMigrations: Migration[] = []
+
+  for (const definition of definitions) {
+    switch (definition.name) {
+      case MigrationName.GLOBAL_INFO_SYNC_USERS: {
+        // only needed in cloud
+        if (!env.SELF_HOSTED) {
```

Comment:
NAB: Might be easier to add this stuff to the definition rather than check for the env type in here.

Reply:
agreed, it would be nice to have a condition in the def itself. Will consider that for future improvements to get this one out